Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -423,6 +423,8 @@ wxMenu* MainMenuBar::CreateSymbolsMenu() const
       IDM_SCAN_SIGNATURES, _("&Signature Database"),
       _("Recognise standard functions from Sys/totaldb.dsy, and use generic zz_ "
         "names for other functions."));
+  generate_symbols_menu->Append(IDM_SCAN_RSO, _("&RSO Modules"),
```

Comment:
Nope, my mistake, I thought the parent menu instance handles all the accelerators. It's only entries in the same menu instance that shouldn't have duplicate accelerators. Sorry about that; my initial comment can be disregarded.